### PR TITLE
Replace slate-editor with DraftJS

### PR DESCRIPTION
**Pull request by niwinz**
https://tree.taiga.io/project/penpot/us/1346

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -12,6 +12,8 @@
 - Duplicate and move files and projects [Taiga #267](https://tree.taiga.io/project/penpot/us/267)
 - Import SVG will create Penpot's shapes
 - Improve french translations [#731](https://github.com/penpot/penpot/pull/731)
+- Replace Slate-Editor with DraftJS [#752](https://github.com/penpot/penpot/pull/752)
```
Comment (hirunatan) [inline CHANGES.md]: Woudn't it be better to link to the US? https://tree.taiga.io/project/penpot/us/1346

File: frontend/src/app/util/text_editor_impl.js [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+  const startOffset = selection.getStartOffset();
+  const block = content.getBlockForKey(startKey);
+  return block.getEntityAt(startOffset);
+}
+
+/**
+ * Copyright (c) UXBOX Labs SL
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ * @license MIT License <https://opensource.org/licenses/MIT>
+ */
```
Comment (hirunatan) [inline frontend/src/app/util/text_editor_impl.js]: why this copyright block here?

Comment (niwinz) [reply frontend/src/app/util/text_editor_impl.js:187]: I link it on the PR :D

File: CHANGES.md (hunk 1 above)
Comment (niwinz) [reply CHANGES.md]: I link it on the PR description

**Pull request closed without merging**